### PR TITLE
Loan Status Prediction using Neural Networks

### PR DESCRIPTION
**Pull request by Srijansarkar17**
## Related Issue
Improves the previous Loan Status Prediction and added my name to contributor list

## Description
- Used Neural Network for training
- Improved accuracy to 87% on training set and 92% on validation set
- Added data visualization

## Type of PR

- [ ] Bug fix
- [X] Feature enhancement
- [X] Documentation update
- [ ] Other (specify): _______________

## Screenshots / videos (if applicable)
[Attach any relevant screenshots or videos demonstrating the changes]
Screenshot 2024-05-31 201823
Screenshot 2024-05-31 201910
Screenshot 2024-05-31 201959

## Checklist:
- [X] I have performed a self-review of my code
- [X] I have read and followed the Contribution Guidelines.
- [X] I have tested the changes thoroughly before submitting this pull request.
- [X] I have provided relevant issue numbers, screenshots, and videos after making the changes.
- [X] I have commented my code, particularly in hard-to-understand areas.

## Additional context:
[Include any additional information or context that might be helpful for reviewers.]

### CODE REVIEW

**Review by Akshat111111 — APPROVED** [review]
Good work!
